.github/workflows/migrate.yml

```diff
@@ -54,7 +59,6 @@ jobs:
 
       - name: Upgrade client to 3.1
```

Comment:
This should probably be renames to 3.x. It would be good to save ourselves the work of renaming this every time we forward merge.

Atm this has resulted in:
https://github.com/juju/juju/blob/3.2/.github/workflows/migrate.yml

Reply:
done